BearMyBanner/BattleBannerAssignBehaviour.cs

```diff
@@ -12,33 +11,26 @@ namespace BearMyBanner
 {
     public class BattleBannerAssignBehaviour : MissionLogic
```

Comment:
I moved entire logic out of this class - what to give to someone etc. Thanks to this it follows SRP way better. Now this class handles only game-specific operations